src/main/java/nextstep/security/authorization/AuthorizationFilter.java

```diff
@@ -0,0 +1,37 @@
+package nextstep.security.authorization;
+
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.ServletRequest;
+import jakarta.servlet.ServletResponse;
+import jakarta.servlet.http.HttpServletRequest;
+import nextstep.security.authentication.Authentication;
+import nextstep.security.context.SecurityContextHolder;
+import nextstep.security.request.MvcRequestMatcher;
+import nextstep.security.request.RequestMatcherEntry;
+import org.springframework.http.HttpMethod;
+import org.springframework.web.filter.GenericFilterBean;
+
+import java.io.IOException;
+import java.util.List;
+
+public class AuthorizationFilter extends GenericFilterBean {
+
+    private final RequestAuthorizationManager authorizationManager;
```

Comment:
```suggestion
    private final AuthorizationManager authorizationManager;
```

구현체가 아닌 인터페이스에 의존하게 변경해보면 어떨까요?